Connectors/Revit/Speckle.Connectors.RevitShared/Bindings/RevitSendBinding.cs

```diff
@@ -110,6 +109,11 @@ public List<ICardSetting> GetSendSettings() =>
 
   public async Task Send(string modelCardId)
   {
+    var document = _revitContext.UIApplication?.ActiveUIDocument?.Document;
+    if (document == null)
+    {
+      return;
```

Comment:
We should throw here. I don't like the idea that a function called "Send" can silently returns without actually sending...  

The send failed, the document state was invalid... exception.